Sources/MapboxMaps/Foundation/MapProjection.swift

```diff
@@ -7,6 +7,10 @@
 ///
 /// Mapbox map supports Mercator and Globe projections.
 @_spi(Experimental) public enum MapProjection: Codable, Hashable {
```

Comment:
StyleProjection is Codable to give us the flexibility to switch to a codable-based implementation in the future if we end up adding any config options besides name. Protocol conformances need to be introduced in the same release as the conforming type to avoid breaking changes.